tests/integration/test_celery.py

```diff
@@ -27,14 +27,15 @@ def test_celery_run_simulation(celery_app,
                                client,
                                add_simulators_to_path_variable,
                                modify_tmpdir,
-                               shieldhit_binary_installed):
+                               shieldhit_binary_installed, monkeypatch):
```

Comment:
take a look here:
https://github.com/yaptide/yaptide/blob/288-proper-logging/tests/integration/conftest.py#L68

Are you sure that `monkeypatch.setenv` modifies only enviroment in this particular function and not leaving modified state in the subsequent ones ?